src/main/main.js

```diff
@@ -0,0 +1,174 @@
+import path from 'path';
+
+import {
+  app,
+  BrowserWindow,
+  screen,
+  nativeTheme,
+  Menu,
+  ipcMain,
+} from 'electron'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
This warning is actually about electron being absent from package.json `dependencies`. It's under `devDependencies` instead. Eslint thinks that's wrong since we're importing it here, but electron-builder wants it only listed in devDeps. Otherwise I think we'd probably end up distributing two electron modules in production.

Reply:
Oh weird, thanks for the explanation.